examples/gno.land/r/demo/boards/README.md

```diff
@@ -3,134 +3,132 @@ constructed by Gno onto a smart contract hosted on the data Realm
 name ["gno.land/r/boards"](https://gno.land/r/boards/)
 ([github](https://github.com/gnolang/gno/tree/master/examples/gno.land/r/boards)).
 
-
-
-## Build `gnokey`, create your account, and interact with Gno.
+## Build `gnokey`, create your account, and interact with Gno
 
 NOTE: Where you see `--remote gno.land:36657` here, that flag can be replaced
 with `--remote localhost:26657` for local testnets.
 
-### Build `gnokey`.
+### Build `gnokey`
 
 ```bash
-git clone git@github.com:gnolang/gno.git
+git clone https://github.com/gnolang/gno.git
 cd ./gno
-make
+make install_gnokey
```

Comment:
For public tutorials, it's preferable to use `make build*` commands. This allows users to run the `./build/XXX` binaries directly without relying on a specific `$PATH` configuration.